packages/hoppscotch-backend/src/infra-config/helper.ts

```diff
@@ -54,6 +64,125 @@ export async function loadInfraConfiguration() {
   }
 }
 
+/**
+ * Read the default values from .env file and return them as an array
+ * @returns Array of default infra configs
+ */
+export async function getDefaultInfraConfigs(): Promise<
```

Comment:
I think this function can be inside `infra-config.service.ts` file itself, was there particular reason we are moving it here?

Reply:
There is a function `isInfraConfigTablePopulated()` in helper.ts that needs `getDefaultInfraConfigs()`. Since, `getDefaultInfraConfigs()` was in infra-config.service.ts, could not able call from `isInfraConfigTablePopulated()` (from helper file without creating object of service class)

Reply:
Alright, understood.